src/cloudant/client.py

```diff
@@ -87,6 +87,7 @@ def __init__(self, user, auth_token, admin_party=False, **kwargs):
         self.server_url = kwargs.get('url')
         self._client_user_header = None
         self.admin_party = admin_party
+        self.decoder = json.JSONDecoder  # not currently configurable
```

Comment:
Since this already caused some confusion, perhaps append `at client scope` or similar.